webapp/tests/karma/ts/services/enketo.service.spec.ts

```diff
@@ -1470,4 +1471,52 @@ describe('Enketo service', () => {
     });
   });
 
+  describe('renderContactForm', () => {
+    beforeEach(() => {
+      service.setFormTitle = sinon.stub();
+      dbGetAttachment.resolves('<form/>');
+    });
+
+    const noop = () => {};
```

Comment:
I'm not sure the meaning of `noop`, I had a question mark when I read it the first time, maybe it's clearer if it's called something like `callbackMock` 🤔 or just use sinon.stub() in the editedListener and valuechangeListener

Reply:
I wasn't sure how to name this function, it's just a placeholder for the two callbacks `editedListener` and `valuechangeListener`. I think it's best to only use `sinon.stub()` for functions we want to mock the function's behavior, I'm renaming the function name to make its purpose clearer